### PR TITLE
DOC-2863 update CV tooltips

### PR DESCRIPTION
**Pull request by SudheendraKatte**
Hi @ravilach , please review and approve.

### CODE REVIEW

**Comment by ravilach** [comment]
LGTM.

**Review by ravilach — APPROVED** [review]

File: src/Tooltip/TooltipDictionary.yaml [hunk 1]
```diff
@@ -1472,22 +1472,22 @@ cvData_timeout:
         [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_2_enter_a_name_and_timeout)
     width: "400"
 cvData_spec.type:
-    content: In Continuous Verification Type, select a type that matches your deployment strategy. Rolling, Canary, Blue Green, and Load Test simply refer to the strategy in the stage. [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_3_select_a_continuous_verification_type)
+    content: In Continuous Verification Type, select a type that matches your deployment strategy. Rolling, Canary, Blue Green, and Load Test simply refer to the strategy in the stage. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#continuous-verification-type)
```
Comment (aimurphy) [inline src/Tooltip/TooltipDictionary.yaml:1475]:
```suggestion
    content: In Continuous Verification Type, select a type that matches your deployment strategy. Rolling, Canary, Blue Green, and Load Test refer to the strategy in the stage. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#continuous-verification-type)
```

File: src/Tooltip/TooltipDictionary.yaml [hunk 2]
```diff
@@ -1472,22 +1472,22 @@ cvData_timeout:
...
     width: "400"
 cvData_spec.monitoredServiceRef:
-    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click Click to autocreate a monitored service. If the stage Service or Environment settings are Runtime Inputs, the Monitored Service and Health Sources settings will show up in the Runtime Input settings when you run the Pipeline. [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_4_create_a_monitored_service)
+    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click to autocreate a monitored service. If the stage Service or Environment settings are Runtime Inputs, the Monitored Service and Health Sources settings will show up in the Runtime Input settings when you run the Pipeline.
```
Comment (aimurphy) [inline src/Tooltip/TooltipDictionary.yaml:1478]:
```suggestion
    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click to autocreate a monitored service. If the stage's Service or Environment settings are runtime inputs, then the Monitored Service and Health Sources settings appear in the runtime input settings when you run the pipeline.
```

This is unclear: "In Monitored Service, click to autocreate a monitored service" What do you click?

File: src/Tooltip/TooltipDictionary.yaml [hunk 3]
```diff
@@ -1472,22 +1472,22 @@ cvData_timeout:
...
     width: "400"
 cvData_spec.spec.sensitivity:
-    content: Choose the level of sensitivity. It is used as an input for the sensitivity in the Verification stage. If you set the sensitivity to high, even small anomalies will be treated as verification failures. [Learn More](https://developer.harness.io/docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-the-verify-step/#step_6_select_sensitivity)
+    content: Choose the level of sensitivity. It is used as an input for the sensitivity in the Verification stage. If you set the sensitivity to high, even small anomalies will be treated as verification failures. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#sensitivity)
```
Comment (aimurphy) [inline src/Tooltip/TooltipDictionary.yaml:1481]:
```suggestion
    content: Choose the level of sensitivity. It is used as an input for the sensitivity in the Verification stage. If you set the sensitivity to high, even small anomalies are treated as verification failures. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#sensitivity)
```

File: src/Tooltip/TooltipDictionary.yaml [hunk 4]
```diff
@@ -1472,22 +1472,22 @@ cvData_timeout:
...
     width: "400"
 cvData_spec.spec.duration:
-    content: In Duration, select the number of data points Harness uses. If you enter 10 minutes, Harness will take the first 10 minutes worth of the log/APM data points and analyze it. The recommended Duration is 10 min for logging providers and 15 min for APM and infrastructure providers. [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_7_select_duration)
+    content: In Duration, select the number of data points Harness uses. If you enter 10 minutes, Harness will take the first 10 minutes worth of the log/APM data points and analyze it. The recommended Duration is 10 min for logging providers and 15 min for APM and infrastructure providers. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#duration)
```
Comment (aimurphy) [inline src/Tooltip/TooltipDictionary.yaml:1484]:
```suggestion
    content: Select the number of data points Harness uses. For example, if you enter 10 minutes, Harness analyzes the first 10 minutes worth of the log/APM data points. The recommended duration is 10 minutes for logging providers and 15 minutes for APM and infrastructure providers. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#duration)  
```

File: src/Tooltip/TooltipDictionary.yaml [hunk 5]
```diff
@@ -1472,22 +1472,22 @@ cvData_timeout:
...
+    content: In Duration, select the number of data points Harness uses. If you enter 10 minutes, Harness will take the first 10 minutes worth of the log/APM data points and analyze it. The recommended Duration is 10 min for logging providers and 15 min for APM and infrastructure providers. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#duration)  
     width: "400"
 cvData_spec.spec.trafficsplit: ""
 cvData_spec.spec.deploymentTag: |-
-    Use a [Harness expression](https://developer.harness.io/docs/platform/Variables-and-Expressions/harness-variables) to reference the artifact in the stage Service settings.
+    Use a [Harness expression](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/) to reference the artifact in the stage Service settings.
```
Comment (aimurphy) [inline src/Tooltip/TooltipDictionary.yaml:1488]:
```suggestion
    Use a [Harness expression](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/) to reference the artifact in the stage's Service settings.
```

**Review by aimurphy — APPROVED** [review]
approved with some suggestions

**Pull request merged**